### ISSUE TITLE
Unexpected Invalid variable  "media" (less/no-duplicate-variables)

### ISSUE THREAD

**Issue by binary64** [ssivanatarajan/stylelint-less#4]
I have lots of nested @media blocks.

Should I restructure my LESS, or could your plugin support this config? :-

```json5
    "less/no-duplicate-variables": [true, {
      "ignore": ["@"]
    }],
```

**Comment by ssivanatarajan** [comment]
@binary64  could you please share the code snippet and configuration to reproduce the issue?

**Comment by GarethWelch** [comment]
@ssivanatarajan Media queries nested in anything appear to exhibit this error for me.

e.g.
```less
.content {
  @media (min-width: 768px) {
    width: 700px;
  }
}
```

**Comment by ssivanatarajan** [comment]
@binary64 & @GarethWelch . Thanks for reporting the issue ,it has been fixed in 1.0.5
